cves/kernel/CVE-2016-0823.yml

```diff
@@ -84,14 +89,12 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
+- commit: a06db751c321546e5563041956a57613259c6720
+  note: | 
+    This commit makes it so that the pagemap is readable to users safely.
+    Previous commit only allowed CAP_SYS_ADMIN to use it.
 - commit: ab676b7d6fbf4b294bf198fb27ade5b0e865c7ce
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+  note: Manually confirmed
 vcc_instructions: |
```

Comment:
This is about the vccs below (Github is being painful). You should check the commits and change the notes to manually confirmed just to make sure these are correct.